service/src/main/kotlin/fi/espoo/evaka/invoicing/domain/FeeDecisions.kt

```diff
@@ -73,6 +73,8 @@ data class FeeDecision(
     override fun isEmpty(): Boolean = this.children.isEmpty()
     override fun annul() = this.copy(status = FeeDecisionStatus.ANNULLED)
     override fun withChildren(children: List<FeeDecisionChild>) = this.copy(children = children)
+
+    fun containsChild(id: ChildId): Boolean = this.children.any { it.child.id == id }
```

Comment:
Is this used anywhere?

Reply:
Not anymore :sweat_smile: It was part of some old refactoring that I decided to revert